packages/opentelemetry-propagator-b3/README.md

```diff
@@ -95,57 +93,6 @@ api.propagation.setGlobalPropagator(
 );
 ```
 
-### B3SinglePropagator
-
-If a distributed system only needs support for the b3 single-header
-encoding it can use the `B3SinglePropagator` directly.
-
-Example usage:
-
-```javascript
-const api = require('@opentelemetry/api');
-const { B3SinglePropagator } = require('@opentelemetry/propagator-b3');
-
-api.propagation.setGlobalPropagator(new B3SinglePropagator());
-```
-
-### B3MultiPropagator
-
-If a distributed system only needs support for the b3 multi-header
-encoding it can use the `B3MultiPropagator` directly.
-
-Example usage:
-
-```javascript
-const api = require('@opentelemetry/api');
-const { B3MultiPropagator } = require('@opentelemetry/propagator-b3');
-
-api.propagation.setGlobalPropagator(new B3MultiPropagator());
-```
-
-### CompositePropagator
```

Comment:
is CompositePropagator not allowed anymore, as with regards to possibility I think it should still be possible ?

```
const api = require('@opentelemetry/api');
const { B3Propagator } = require('@opentelemetry/propagator-b3');
api.propagation.setGlobalPropagator(
  new CompositePropagator({
    propagators: [
      new B3Propagator(), 
      new B3Propagator({ injectEncoding: B3InjectEncoding.MULTI_HEADER })
    ],
  })
);
``` 
?

Reply:
By default it already extracts both. Is there a usecase where we need to also inject both? Of course you can always use a composite just like you can with any propagator, but keeping it documented feels unnecessary for such a niche usecase.

Reply:
It's still possible. I don't think it's a common use case, but we can document it.